### PR TITLE
Added product details #30

### LINKED ISSUE #30: List product details on the product page
Please fetch the product details from the `/product/{id}` api while viewing `#/product/:id`, and display them.
Please use the bootstrap grid for the columns.

### PR DESCRIPTION
**Pull request by krisztianm**
Fixes #30

### CODE REVIEW

File: app/src/productDetailsController.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+(function () {
+    angular
+        .module('webshopModule')
+        .controller('ProductDetailsCtrl', ProductDetailsController);
+
+    function ProductDetailsController($http, $scope) {
+            $http({
+                method: 'GET',
+                url: 'http://localhost:1337/product/{id}'
```
Comment (zsoltgyongyosi) [inline app/src/productDetailsController.js]: We have to provide an explicit `id` to the backend. Please check out the `$routeParams` service.

File: app/templates/productDetails.html [hunk 2]
```diff
@@ -1,15 +1,11 @@
...
-                <li>Nullam eu varius lectus</li>
-                <li>Proin rhoncus ac metus non egestas</li>
-            </ul>
-        </td>
-    </tr>
-</table>
+<div class="col-md-4">
+    <img src="http://lorempixel.com/120/40/abstract/" alt="Product Image">
+</div>
+<div class="col-md-8" ng-repeat="product in products">
```
Comment (zsoltgyongyosi) [inline app/templates/productDetails.html]: We only have 1 product in this case.

File: app/templates/productDetails.html [hunk 3]
```diff
@@ -1,15 +1,11 @@
...
-                <li>Lorem ipsum dolor sit amet</li>
-                <li>Consectetur adipiscing elit</li>
-                <li>Nullam eu varius lectus</li>
-                <li>Proin rhoncus ac metus non egestas</li>
-            </ul>
-        </td>
-    </tr>
-</table>
+<div class="col-md-4">
+    <img src="http://lorempixel.com/120/40/abstract/" alt="Product Image">
```
Comment (zsoltgyongyosi) [inline app/templates/productDetails.html]: Please ensure that this image fills the column via CSS.

**Review by zsoltgyongyosi — CHANGES_REQUESTED** [review]

File: scss/index.scss [hunk 4]
```diff
@@ -20,3 +20,11 @@ footer {
     width: 100%;
     height: 50px;
 }
+
+
+.productDetails {
+    background: url("http://lorempixel.com/120/40/abstract/") center;
```
Comment (zsoltgyongyosi) [inline scss/index.scss]: This one will have to be an `` tag, because it's not a static image.

File: scss/index.scss [hunk 5]
```diff
@@ -20,3 +20,11 @@ footer {
     width: 100%;
     height: 50px;
 }
+
+
+.productDetails {
```
Comment (zsoltgyongyosi) [inline scss/index.scss]: Please use kebab-case for classes.

File: app/src/routing.js [hunk 6]
```diff
@@ -8,11 +8,16 @@
                     controller: 'ProductListCtrl'
                 })
                 .when('/product/:id', {
+
+                    templateUrl: 'templates/productDetails.html',
```
Comment (zsoltgyongyosi) [inline app/src/routing.js:11]: Please remove the duplicate key.

**Review by zsoltgyongyosi — CHANGES_REQUESTED** [review]

File: app/src/routing.js (hunk 6 above)
Comment (krisztianm) [reply app/src/routing.js:11]: @zsoltgyongyosi Could you please give more details about this request?

Comment (zsoltgyongyosi) [reply app/src/routing.js:11]: `templateUrl` is duplicated

**Review by zsoltgyongyosi — APPROVED** [review]

**Pull request merged**